### PR TITLE
RS policy references

### PR DESCRIPTION
**Pull request by rrelledge**
[Staged preview](https://docs.redis.com/staging/preview-policy-ref/rs/references/policies/)

### CODE REVIEW

**Review by cmilesb — APPROVED** [review]

File: content/rs/databases/configure/proxy-policy.md [hunk 1]
```diff
@@ -65,82 +52,59 @@ cluster configuration:
...
+{{<note>}}
+To find the endpoint ID, run [`rladmin status endpoints`]({{<relref "/rs/references/cli-utilities/rladmin/status#status-endpoints">}}).
+{{</note>}}
 
-### Reapply policies after topology changes
+## Reapply policy after topology changes
 
-If you want to reapply the policy after topology changes, such as node restarts,
-failovers and migrations, run this command to reset the policy:
+During regular cluster operations, certain actions (such as automatic migration or automatic failover) change what proxy needs to be bound to what database. When such actions take place, the cluster attempts to automatically change proxy bindings to follow to the current proxy policy.
```
Comment (nermiller) [inline content/rs/databases/configure/proxy-policy.md:75]:
```suggestion
During regular cluster operations, certain actions (such as automatic migration or automatic failover) change which proxy needs to be bound to which database. When such actions take place, the cluster attempts to automatically change proxy bindings to follow the current proxy policy.
```

File: content/rs/references/policies/redis-upgrade-policy.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+toc: "true"
+categories: ["RS"]
+aliases: 
+---
+
+In version 6.2.4, Redis Enterprise Software introduced the Redis database compatibility upgrade policy (`redis_upgrade_policy`). This policy limits the [default Redis version]({{<relref "/rs/references/policies/default-redis-version">}}) when creating or updating databases.
+
+Changes to the upgrade policy do not affect existing databases. The policy is used only when you create a new database, upgrade a database, or change its configuration.
+
+For best results, we recommend changing the policy value only after upgrading to a major release of Redis Enterprise Software.
```
Comment (nermiller) [inline content/rs/references/policies/redis-upgrade-policy.md:16]:
```suggestion
For best results, change the policy value only after upgrading to a major release of Redis Enterprise Software.
```

File: content/rs/references/policies/redis-upgrade-policy.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+## Policy values
+
+| Value | Description |
+|-------|-------------|
+| latest | Limits compatibility to the latest (most recent) version of open source Redis supported by your copy of Redis Enterprise Software (default behavior for Redis Enterprise versions earlier than v6.2.4) |
+| major  | Limits Redis compatibility to major releases (default policy for Redis Enterprise v6.2.4 and later) |
+
+As of v6.2.4, `redis_upgrade_policy` defaults to `major`, which limits Redis database compatibility to the most recent major release.
+
+If you change the policy to `latest`, you need to upgrade Redis Enterprise Software every time there’s a minor release. Further, you’ll need to leave the policy set to `latest` until the next major release of Redis Enterprise Software, which generally happens every 18-24 months.
```
Comment (nermiller) [inline content/rs/references/policies/redis-upgrade-policy.md:27]:
```suggestion
If you change the policy to `latest`, you need to upgrade Redis Enterprise Software with each minor release. Furthermore, leave the policy set to `latest` until the next major release of Redis Enterprise Software, which generally happens every 18-24 months.
```

**Review by nermiller — APPROVED** [review]

File: content/rs/references/cli-utilities/rladmin/tune.md [hunk 4]
```diff
@@ -56,6 +57,7 @@ Redis cluster watchdog supports two preconfigured profiles:
...
 | default_redis_version                  | version number                    | The default Redis database compatibility version used to create new databases.<br/><br/>  The value parameter should be a version number in the form of "x.y" where _x_ represents the major version number and _y_ represents the minor version number.  The final value corresponds to the desired version of Redis.<br/><br/>You cannot set _default_redis_version_ to a value higher than that supported by the current _redis_upgrade_policy_ value. |
+| default_shards_placement | `dense`<br />`sparse` | New databases created in the cluster place shards according to the [default shard placement policy]({{<relref "/rs/references/policies/default-shards-placement">}}).<br />• `dense` - place as many shards as possible on the smallest number of nodes to reduce the latency between the proxy and the database shards (default)<br />•`sparse` - spread the shards across as many nodes in the cluster as possible to spread the traffic across cluster nodes |
```
[the remark this answers is not in the document]
Comment (rrelledge) [reply content/rs/references/cli-utilities/rladmin/tune.md:60]: Yes, using `•` was intentional here. `*` and `-` do not render as bullet points inside these tables. Although, it looks like I can use HTML tags instead, if you would prefer. `<ul><li>dense</li><li>sparse</li></ul>` should render like this, even inside the table:
<ul><li>dense - place as many shards as possible on the smallest number of nodes to reduce the latency between the proxy and the database shards (default)</li><li>sparse - spread the shards across as many nodes in the cluster as possible to spread the traffic across cluster nodes</li></ul>

**Review by nermiller — APPROVED** [review]

**Comment by rrelledge** [comment]
Closing without merging. To be reworked and expanded in a future PR.

**Pull request closed without merging**